demo/mkldnn_quant/README_en.md

```diff
@@ -0,0 +1,181 @@
+- **--debug:** Generate models graph or not. Add this option to generate a series of *.dot files containing model drawings after each conversion step. For the description of DOT format, please refer to [DOT](https://graphviz.gitlab.io/_pages/doc/info/lang.html). To open the `*.dot` file, please use any Graphviz tool available on the system(such as the `xdot` tool on Linux or the `dot` tool on Windows. For Graphviz documentation, see [Graphviz](http://www. graphviz.org/documentation/).
+  
+- **Note:**
+  - The DNNL supported quantizable ops are `conv2d`, `depthwise_conv2d`, `fc`, `matmul`, `pool2d`, `reshape2`, `transpose2`, `scale`, `concat`.
+  - If you set `--op_ids_to_skip`, you only need to pass in all the quantized op ID number you want to keep FP32 type.
+  - Sometimes quantizing all ops does not necessarily result in optimal performance. For example: If an op is a single INT8 op, before and after the op are float32 op, then in order to quantify this op, you need to do quantize first, then run INT8 op, and then dequantize, which may lead to the final performance is not as good as keeping the op fp32 op. If the user has poor performance using the default settings, you can observe whether this model has a separate INT8 op, select different combinations of `ops_to_quantize`, or you can exclude some quantiable op IDs through `--op_ids_to_skip`, and run several times to get the most Good settings.
+
+## 4. Inference
+
+### 4.1 Data preprocessing
+To deploy on the CPU, the data needs to be binary converted first. Run the script as follows to transform the complete ILSVRC2012 val data set. Use `--local` to convert user's own data. Run the following script in the directory where Paddle is located. The script is located on the official website at [full_ILSVRC2012_val_preprocess.py](https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py)
+```
+python Paddle/paddle/fluid/inference/tests/api/full_ILSVRC2012_val_preprocess.py --local --data_dir=/PATH/TO/USER/DATASET/ --output_file=/PATH/TO/SAVE/BINARY/FILE
+```
+
+**Available options in the above command and their descriptions are as follows:**
+- No parameters are set. The script will download the ILSVRC2012_img_val data set and convert it into a binary file.
+- **local:** is set to true, indicating that users will provide their own data
+- **data_dir:** user's own data directory
+- **label_list:** Picture path-Picture category list file, similar to `val_list.txt`
+- **output_file:** The path of the generated binary file.
+- **data_dim:** Pre-process the length and width of the picture. The default value is 224.
```

Comment:
"The length and width for the pre-processed pictures in the resulting binary. The default value is 224."